packages/tuxguitar/PKGBUILD

```diff
@@ -0,0 +1,113 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Stephan Springer <buzo+arch@Lini.de>
+# Contributor: Thanos Apostolou <thanosapostolou@outlook.com>
+# Contributor: Alexandre Moine <alexandre@moine.me>
+# Contributor: Sergej Pupykin <pupykin.s+arch@gmail.com>
+# Contributor: |AhIoRoS| < ahioros@gmail.com >
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgbase=tuxguitar
+pkgname=(tuxguitar tuxguitar-common tuxguitar-gtk2)
+pkgver=1.5.5
+pkgrel=2
+pkgdesc="Multitrack guitar tablature editor and player"
+arch=('x86_64' 'i686' 'armv7h')
```

Comment:
Should this by `any` because the binary is Java?

Reply:
The `tuxguitar-common` package includes platform-specific libs using JNI. This was also [reported by namcap](https://ci.cbix.de/osam-cologne/archlinux-proaudio/40/1/7):
```
tuxguitar-common E: ELF file ('usr/share/tuxguitar/lib/libicedtea-sound.so') found in an 'any' package.
tuxguitar-common E: ELF file ('usr/share/tuxguitar/lib/libtuxguitar-alsa-jni.so') found in an 'any' package.
tuxguitar-common E: ELF file ('usr/share/tuxguitar/lib/libtuxguitar-fluidsynth-jni.so') found in an 'any' package.
tuxguitar-common E: ELF file ('usr/share/tuxguitar/lib/libtuxguitar-jack-jni.so') found in an 'any' package.
tuxguitar-common E: ELF file ('usr/share/tuxguitar/lib/libtuxguitar-oss-jni.so') found in an 'any' package.
```